_posts/blog/2018-10-11-review-of-periodicos-e-literatura.md

```diff
@@ -0,0 +1,27 @@
+
+As mentioned before, there is a conflict in the way that the digital is presented in the narratives about the NDLB. The [*introduction page*](https://bndigital.bn.gov.br/sobre-a-bndigital/apresentacao/https://bndigital.bn.gov.br/sobre-a-bndigital/apresentacao/) considers that digitization exposes the crises of cultural structures related to the book and reading, and then goes into a lengthy discussion of the pitfalls of amassing cultural objects and the techno-market. This appears to affect usability in significant ways such as absence of a search feature, sorting mechanisms or easily identifiable metadata; performing a reiteration of "needing to know what you are looking for" even when there is a different thematic organization of the materials. The website also becomes inaccessible for hours. 
+
+I believe this is partially due to budget constraints, but also the framing of the objects as digital copies of the valuable physical materials. For example, their digitization is used as a justification for limiting access to the physical materials to prevent their deterioration. Regardless, the content of the collection is only limited because of its vastness. Caser da Costa lists the figure of four hundred texts in her master's thesis. The amount of material is sufficient to meet the goals of the project and facilitate research in an area that is gaining traction in [*digital humanities*](http://oceanicexchanges.org/) and [*literary studies*](http://scholarlyediting.org/2013/editions/intro.cwnewspaperpoetry.html). As mentioned before, the effect is also felt in the field of LIS, starting with the example of Caser da Costa and continuing through a number of events and workshops led by coordinators of the collection. Perhaps conceiving itself as a digital collection can help to ameliorate some of these limitations and open up the materials to other approaches and methodologies. 
```

Comment:
I find this final paragraph confusing. I think you're suggesting that the website has not fully exploited the potential of the digital archive to enable new kinds of research, but I'm having trouble following the argument through the various examples that you give. Reorganizing the details in this paragraph will help make your argument easier to follow.